ext/project_data.yaml

```diff
@@ -13,14 +13,6 @@ gem_require_path: 'lib'
 gem_test_files: 'spec/**/*'
 gem_executables: 'hiera'
 gem_default_executables: 'hiera'
-gem_platform_dependencies:
-  x86-mingw32:
-    gem_runtime_dependencies:
-      win32console: '1.3.2'
-      win32-dir: '~> 0.4.8'
-  x64-mingw32:
-    gem_runtime_dependencies:
-      win32-dir: '~> 0.4.8'
 bundle_platforms:
   x86-mingw32: mingw
   x64-mingw32: x64_mingw
```

Comment:
Yeah, thought about that... there's actually a lot of stuff we could pull out here, but I decided to not to scrap all of it for a few reasons:

* The event we need to reintroduce any platform specific bits
* Consistency with other projects
* Its being consumed in Rakefile (when I tested this out I ran the rake task to build gems)

IIRC, to remove `bundle_platforms` necessitates that we update the `Gemfile` to remove the entire section that loads `project_data.yaml` at https://github.com/puppetlabs/hiera/pull/393/files#diff-8b7db4d5cc4b8f6dc8feb7030baa2478R21

If you don't think we care about those things, and we're OK with also gutting relevant portions of the `Rakefile` to update packaging, then we can do that.  I just didn't want to get too carried away just yet...